src/iguana/algorithms/clas12/PhotonGBTFilter/Validator.cc

```diff
@@ -0,0 +1,176 @@
+#include "Validator.h"
+namespace iguana::clas12 {
+
+  REGISTER_IGUANA_VALIDATOR(PhotonGBTFilterValidator);
+
+  void PhotonGBTFilterValidator::Start(hipo::banklist& banks)
+  {
+    // define the algorithm sequence
+    m_algo_seq = std::make_unique<AlgorithmSequence>();
+    m_algo_seq->Add("clas12::PhotonGBTFilter");
+    m_algo_seq->Add("clas12::EventBuilderFilter");
```

Comment:
It would be faster to run `EventBuilderFilter` first
```suggestion
    m_algo_seq->Add("clas12::EventBuilderFilter");
    m_algo_seq->Add("clas12::PhotonGBTFilter");
```

Reply:
As mentioned in a previous comment, the PhotonGBTFilter must be run first because we require all nearest neighbors from the REC::Particle bank to be considered before filtering. Removing particles with prior filters may call the trustworthiness of the GBT output into question.